github/repos.go

```diff
@@ -834,6 +834,11 @@ type Branch struct {
 	Protected *bool             `json:"protected,omitempty"`
 }
 
+type RequiredSignatures struct {
```

Comment:
Please add a godoc-style comment to this exported struct (as a complete sentence with punctuation).